src/modules/components/proposal/proposalDataListItem/proposalDataListItemStructure/proposalDataListItemStructure.test.tsx

```diff
@@ -82,16 +82,42 @@ describe('<ProposalDataListItemStructure/> component', () => {
         expect(screen.getByRole('link', { name: 'You' })).toBeInTheDocument();
     });
 
+    it('renders multiple publishers', () => {
+        const publishers = [
+            { name: 'abc', profileLink: '#', address: '0x123' },
+            { name: 'def', profileLink: '#', address: '0x123' },
+        ];
+
+        render(createTestComponent({ publisher: publishers }));
+
+        publishers.forEach((publisher) => {
+            expect(screen.getByText(publisher.name)).toBeInTheDocument();
+        });
+    });
+
+    it("renders '3+ creators' when the publishers are more than 3", () => {
+        const publishers = [
+            { name: 'abc', profileLink: '#', address: '0x123' },
+            { name: 'def', profileLink: '#', address: '0x123' },
+            { name: 'ghi', profileLink: '#', address: '0x123' },
+            { name: 'jkl', profileLink: '#', address: '0x123' },
+        ];
+
+        render(createTestComponent({ publisher: publishers }));
+
+        expect(screen.getByText('3+ creators')).toBeInTheDocument();
+    });
+
     describe("'approvalThreshold type'", () => {
         it('renders without crashing', () => {
             const testProps: IProposalDataListItemStructureProps = {
                 date: new Date().toISOString(),
-                publisher: { address: '0x123' },
-                publisherProfileLink: '#',
+                publisher: { address: '0x123', profileLink: '#' },
                 status: 'active',
                 summary: 'Example Summary',
                 title: 'Example Title',
                 type: 'approvalThreshold',
+                id: '0x1',
```

Comment:
Let move the testing of id and title properties outside the result type as not dependent by it. We could write it once and do not change both the result type tests every time we add, change, remove a property